### ISSUE TITLE
Missing Authorization header when using FETCH

### ISSUE THREAD

**Issue by Vainock** [crowdin/crowdin-api-client-js#152]
I'm unable to get the client working with FETCH. My code:
```js
import CROWDIN, { HttpClientType } from "@crowdin/crowdin-api-client";

async function a() {
  return (
    await new CROWDIN(
      { token: API_TOKEN},
      { httpClientType: HttpClientType.FETCH }
    ).projectsGroupsApi.getProject(PROJECT_ID)
  ).data;
}

console.log(a())
```

When executing this code in the browser it successfully sends a request to `https://api.crowdin.com/api/v2/projects/PROJECT_ID` but returns a `401`, that's because it's missing the `Authorization` header.
(The API key and project id I'm providing are correct, but that seems not to be the issue.)

**Comment by Olena1234** [comment]
Hi there! Could you please provide us with the link to the project?

**Comment by Vainock** [comment]
I'm using a private project: https://crowdin.com/project/crowdin-sync-testing. Thank you for your help!

**Comment by andrii-bodnar** [comment]
Hi @Vainock!

Did you try to follow the code samples from the Readme?

```js
import { ProjectsGroups, HttpClientType } from '@crowdin/crowdin-api-client';

const credentials: Credentials = {
  token: 'personalAccessToken'
};

const projectsGroupsApi = new ProjectsGroups(credentials, {
  httpClientType: HttpClientType.FETCH
});

// get all projects
projectsGroupsApi
  .withFetchAll()
  .listProjects()
  .then(projects => console.log(projects))
  .catch(error => console.error(error));

```

**Comment by Vainock** [comment]
Yes, I've just tried this example and the result is exactly the same. Btw, this example from the README is missing the import of `Credentials`.

**Comment by andrii-bodnar** [comment]
@Vainock please try older versions of the JS API Client just in case. Also, please make sure you are using a valid access token and have permission to view details of the specified project.

**Comment by Vainock** [comment]
https://codesandbox.io/embed/friendly-meadow-v4ne8w

I've tried it again with an online code sandbox and the result is still the same, if you inspect the site and go to the Networks tab, you will see the failed request. I've entered my PAT, which has access to my account because I'm able to send this request via Postman. The problem are not my permissions, the API client just doesn't add the `Authorization:` header to the request.
